db/dataset.py

```diff
@@ -509,3 +509,27 @@ def update_class(dataset_id, class_name, class_data):
             params["id"] = clsid
 
             connection.execute(update_query, params)
+
+
+def check_recording_in_dataset(dataset_id, mbid):
+    """Check whether an MBID is in a given dataset.
+    Args:
+        dataset_id: ID of a dataset
+        mbid: MBID to be checked
+    Returns:
+        True if an MBID appears anywhere in a dataset, False otherwise
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT dataset_class.id
+              FROM dataset_class
+              JOIN dataset_class_member
+                ON dataset_class_member.class = dataset_class.id 
+             WHERE dataset_class.dataset = :dataset_id
+               AND dataset_class_member.mbid = :mbid
+        """), {"dataset_id": dataset_id, "mbid": mbid})
+        
+        if result.rowcount > 0:
```

Comment:
Just a `return result.rowcount > 0` would look better here, imo :)

Reply:
lucky you didn't see what I had before:
```
count = result.rowcount
if count > 0.....
```
in general I prefer it a bit more explicit, but I'm OK either way